scene/main/instance_placeholder.cpp

```diff
@@ -160,13 +160,13 @@ void InstancePlaceholder::set_value_on_instance(InstancePlaceholder *p_placehold
 		// but the instance wants a Node Array.
 	}
 
+	Ref<Resource> resource = p_set.value;
```

Comment:
This should be in the `case Variant::Type::NIL`, since it's only used there.

Reply:
Fixed.

I tried to do that at the first but i received an error about jumping variable initialization so i thought that wasn't allowed (i have little knowledge about c++) but i did a fast research and i just need to enclose the case with curly brackets.